pkg/controller/servicemesh/memberroll/controller.go

```diff
@@ -379,6 +382,13 @@ func (r *ReconcileMemberList) Reconcile(request reconcile.Request) (reconcile.Re
 	return reconcile.Result{}, kialiErr
 }
 
+func (r *ReconcileMemberList) findConfiguredNamespaces(meshNamespace string) (corev1.NamespaceList, error) {
+	list := corev1.NamespaceList{}
+	labelSelector := map[string]string{common.MemberOfKey: meshNamespace}
```

Comment:
Why use the label and not e.g. the Status.ConfiguredMembers?

Reply:
Before this PR, the operator determined configured members in two ways (in two different places):
- lists namespaces with the label
- gets namespaces from `spec.members`.

This PR ensures that the same method is used in both places. 

Using `status.configuredMembers` might result in the operator not cleaning up a namespace if the SMMR object is stale and is missing one of the namespaces. But you've now got me thinking. Even the namespaces themselves can be stale, so you can actually run into the same problem.

Reply:
> This PR ensures that the same method is used in both places.

^^ I think that alone makes sense. We can see if we want to change general behaviour later on.